sos/report/plugins/coredump.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2023 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, PluginOpt
+
+
+class Coredump(Plugin, RedHatPlugin):
```

Comment:
Maybe add ubuntu too, since coredumpctl is available in ubuntu from the systemd-coredump package

Reply:
ack, will add Ubuntu as well

Reply:
I should add DebianPlugin as well, I imagine, right?

Reply:
Yes I think that would be a good idea.

Reply:
Worth to stay with `[|RedHat|Ubuntu|Debian]Plugin`, or directly with `[|Independent]Plugin` (which contains bit more)?

Reply:
I was thinking about using this approach, that we use in other plugins:

`class RedHatCoredump(Coredump, RedHatPlugin):

    packages = ('systemd-udev',)

    def setup(self):
        super(RedHatCoredump, self).setup()


class DebianCoredump(Coredump, DebianPlugin, UbuntuPlugin):

    packages = ('systemd-coredump',)

    def setup(self):
        super(DebianCoredump, self).setup()
`

Because the packages are different. But if 'packages' is 'any' instead of 'all', then I can simplify it with Independent plugin and list all packages in the same line.

Reply:
`packages` is indeed 'any'. `IndependentPlugin` makes sense here.